### PR TITLE
Logs panel: Mark up for translations

### LINKED ISSUE #74305: i18n: Mark up Logs Panel for translations
### Requirements ###

Mark up https://github.com/grafana/grafana/blob/v10.1.0/public/app/features/explore/Logs components for translations

### Background

Please follow #70957 for more info and guidelines.

### PR DESCRIPTION
**Pull request by siddhikhapare**
**What is this feature?**

Mark up Logs Panel for translations

**Why do we need this feature?**

Using translation for log panels allows to internationalize and localize the text and messages within  log-related components.

**Who is this feature for?**

For all

**Which issue(s) does this PR fix?**:

Fixes #74305 

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by ivanahuckova — COMMENTED** [review]
Thanks for your contribution! 🚀 🧡 Before we proceed with the review, would you mind removing the modification made in `public/app/plugins/datasource/influxdb/components/editor/query/influxql/visual/TagsSection.tsx`? We'd like to keep the changes in this pull request confined to the log panel components only.

**Comment by siddhikhapare** [comment]
@ivanahuckova Sure. I will update it.

**Review by matyax — CHANGES_REQUESTED** [review]
Thank you for your contribution @siddhikhapare !

There seem to be a few things to change and fix, and also to revert. Please double check that your PR follows what's defined in https://github.com/grafana/grafana/blob/main/contribute/internationalization.md , and if possible double check with a local Grafana instance that the changes are not creating visual bugs.

Here you can find instructions to set up a local Grafana development instance. https://github.com/grafana/grafana/blob/main/contribute/developer-guide.md

File: public/app/features/explore/Logs/LogsMetaRow.tsx [hunk 1]
```diff
@@ -113,24 +118,27 @@ export const LogsMetaRow = React.memo(
...
             </Button>
           </Tooltip>
         ),
       });
     }
     const downloadMenu = (
       <Menu>
-        <Menu.Item label="txt" onClick={() => downloadLogs(DownloadFormat.Text)} />
-        <Menu.Item label="json" onClick={() => downloadLogs(DownloadFormat.Json)} />
+        <Menu.Item label={t('logs-meta-row.txt-label', 'txt')} onClick={() => downloadLogs(DownloadFormat.Text)} />
```
[the remark this answers is not in the document]
Comment (siddhikhapare) [reply public/app/features/explore/Logs/LogsMetaRow.tsx:140]: @matyax Thanks for your feedback. I am going to add JSON translations. I am working on it. I want to ask that how to translate other languages for logs? I added locally in en (US) format. I used this: https://github.com/grafana/grafana/blob/813ebf988d354bf23e981e00c9c8458995774c21/contribute/internationalization.md

File: public/app/features/explore/Logs/LiveLogs.tsx [hunk 2]
```diff
@@ -162,19 +163,21 @@ class LiveLogs extends PureComponent<Props, State> {
...
-            Clear logs
+            <Trans i18nKey="live-logs.clear-logs">Clear logs</Trans>
           </Button>
           <Button icon="square-shape" variant="secondary" onClick={this.props.stopLive} className={styles.button}>
-            Exit live mode
+            <Trans i18nKey="live-logs.exit-live-mode">Exit live mode</Trans>
           </Button>
           {isPaused ||
             (this.rowsToRender().length > 0 && (
-              <span>
```
[the remark this answers is not in the document]
Comment (siddhikhapare) [reply public/app/features/explore/Logs/LiveLogs.tsx:175]: @matyax I will update it.

File: public/app/features/explore/Logs/LogsMetaRow.tsx [hunk 3]
```diff
@@ -113,24 +118,27 @@ export const LogsMetaRow = React.memo(
 
     // Add unescaped content info
     if (hasUnescapedContent) {
+      const tooltipContent = t(
```
[the remark this answers is not in the document]
Comment (siddhikhapare) [reply public/app/features/explore/Logs/LogsMetaRow.tsx:121]: @matyax I will remove it.

File: public/app/plugins/datasource/influxdb/components/editor/query/influxql/visual/TagsSection.tsx [hunk 4]
```diff
@@ -81,11 +80,7 @@ const Tag = ({ tag, isFirst, onRemove, onChange, getTagKeyOptions, getTagValueOp
         loadOptions={getTagKeySegmentOptions}
         onChange={(v) => {
           const { value } = v;
-          if (value === undefined) {
```
[the remark this answers is not in the document]
Comment (matyax) [reply public/app/plugins/datasource/influxdb/components/editor/query/influxql/visual/TagsSection.tsx:84]: Just revert the changes 👍

**Comment by siddhikhapare** [comment]
@matyax @ivanahuckova I found error regarding memory limit while running yarn start in my VM so I have created new setup and  new PR related this log panel issue. https://github.com/grafana/grafana/pull/75262  I value your opinion, so please review this when you can. I will close this PR. Thank you. images

**Comment by matyax** [comment]
Sounds good @siddhikhapare !

**Pull request closed without merging**